### ISSUE TITLE
Background Color Issues

### ISSUE THREAD

**Issue by Sramos9191** [Shift3/new-sierra#3]
OS X El Capitan
Version 10.11.3

Firefox 44.0.1

These are the color codes from the Style Guide
Frost Blue: #076a8a
Teal: #429895

The background color appearing on the website:
Frost Blue: #186b88
Teal: #5aa1b6

**Comment by KWenn** [comment]
I am assuming that this is only in reference to the front page which is a different styling than the rest of the pages which have an image as the header with a white background color. I lifted the color directly off of the psd file. Please specify with a screen shot?

**Comment by Sramos9191** [comment]
Yes, this is only in reference to the front page AND the colors do match the psd however the psd itself doesn't match the style guide's specs for the Frost Blue and Teal.

**Comment by KWenn** [comment]
Sorry, still a bit confused on this one. So if we are referring to the boxes, then I am using the correct colors but not hex, I am using rgba to handle the opacity. I double checked them agains the style guide and psd, they are correct. If I am missing something, please explain.

.boxone{
	background-color:rgba(66, 152, 149, 0.8);
	color:#fff;
	}
	
.boxtwo{
	background-color:rgba(80, 108, 67, 0.8);
	color:#fff;
	}
	
.boxthree{
	background-color:rgba(3, 32, 74, 0.8);
	color:#fff;
	}
	
.boxfour{
	background-color:rgba(7, 106, 138, 0.8);
	color:#fff;
	}

**Comment by Sramos9191** [comment]
No, we are not referring to the boxes. 

![screen shot 2016-02-13 at 10 38 59 pm](https://cloud.githubusercontent.com/assets/7436666/13032336/9d98ba80-d2a2-11e5-9356-e937b6f40017.png)
if you scroll down the page towards the bottom, the colors that are the background colors on the page are the colors that don't match the style guide.

**Comment by KWenn** [comment]
Not trying to be a pain here, but the colors don't match in the psd and I see absolutely no reference to those colors in the style guide. Are you sure these are being changed correctly? Here is a sample of the colors you are referencing to change to on the right with the current colors on the left. Can you share a sample of where the colors or referenced on the style guide?

![sample](https://cloud.githubusercontent.com/assets/9420221/13032354/06bd6d7a-d2a4-11e5-99f7-ce8e59048917.jpg)

**Comment by Sramos9191** [comment]
You're not being a pain, this is good. I was only going off of the hex numbers for the Frost Blue and Teal. It doesn't specify if the background colors are intended to follow those specs, it was my assumption. They do match the psd's but not the Style Guide. I'll check with Miguel on this one. We can leave it be for now.

**Comment by KWenn** [comment]
ok, great. Thanks.

**Comment by Sramos9191** [comment]
@malarconjr5 the psd matches what he has but the colors on the style guide don't match the psd, should it be left because it matches the psd's or changed because it doesn't match the style guide ?
